src/service/telemetryHandlerService.ts

```diff
@@ -244,8 +244,8 @@ export class TelemetryHandlerService {
                 break;
         }
         return {
-            deployTelemetryEvent: deployTelemetryEvent,
-            deployPerformanceTelemetryEvent: deployPerformanceTelemetryEvent,
+            deployTelemetryEvent,
```

Comment:
Just wondering, is this still accessible the same way?

Reply:
Yes, this typing is not necessary. Change was made by linting.

Reply:
fascinating, didn't know it worked like that. thanks :D